include/pmacc/mappings/kernel/AreaMappingMethods.hpp

```diff
@@ -49,7 +49,7 @@ namespace pmacc
         }
 
         template<class Base>
-        HDINLINE static DataSpace<DIM> getBlockIndex(
+        HDINLINE static DataSpace<DIM> getSuperCellIndex(
```

Comment:
Note: this was named incorrectly. This function calculates supercell index using block index. So i renamed it. However it is only used internally in the respective mapper. Same for other similar functions changed in this PR